pkg/api/types.go

```diff
@@ -230,3 +241,37 @@ const (
 func NewDockerNetworkModeContainer(id string) DockerNetworkMode {
 	return DockerNetworkMode(DockerNetworkModeContainerPrefix + id)
 }
+
+// PullPolicy specifies a type for the method used to retrieve the Docker image
+type PullPolicy string
+
+// String implements the String() function of pflags.Value so this can be used as
+// command line parameter.
+// This will also default to "if-not-present" if this parameter is not set.
+func (p *PullPolicy) String() string {
+	if len(string(*p)) == 0 {
```

Comment:
Hmmm... not sure, I'd rather see when it's actually empty and not default it in some hidden place.